src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -180,26 +176,25 @@ public MainMenu() {
 				Server.getInstance().login(myUser.getUserName(), myUser.getPasswd());
 				try {
 					Thread.sleep(500);
-				} catch (InterruptedException interruptedException) {
-					LOGGER.error(interruptedException.getMessage(), interruptedException);
+				} catch (InterruptedException e) {
+					e.printStackTrace();
```

Comment:
correct form is with interruptedException and LOGGER.error

Reply:
restored current informatici version